package.json

```diff
@@ -38,25 +38,33 @@
         "@testing-library/jest-dom": "^4.2.4",
         "@testing-library/react": "^9.5.0",
         "@testing-library/user-event": "^7.2.1",
+        "@types/d3-timer": "^1.0.9",
         "@types/firebase": "^3.2.1",
+        "@types/fscreen": "^1.0.1",
         "@types/jest": "^26.0.15",
-        "@types/node": "^14.14.6",
+        "@types/node": "^12.12.26",
```

Comment:
I used the same version of the package used in [Twilio React Video App](https://github.com/twilio/twilio-video-app-react) repository, in order to avoid having to fix all `ts` errors